frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -44,7 +45,7 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
                 {children}
               </MultiColumnFieldRow>
             ))}
-          {!isReadOnly && (
+          {!isReadOnly && !disableAddValues && (
```

Comment:
Work with @karthikjeeyar who is doing the same thing here: https://github.com/openshift/console/pull/3486